cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -93,11 +97,36 @@ func newDevicePlugin(pciDriverDir, pciDeviceDir string, maxDevices int, kernelVf
 		pciDeviceDir:    pciDeviceDir,
 		kernelVfDrivers: kernelVfDrivers,
 		dpdkDriver:      dpdkDriver,
+		scanTicker:      time.NewTicker(scanPeriod),
+		scanDone:        make(chan bool, 1),
+	}
+}
+
+func (dp *DevicePlugin) setupDeviceIDs() error {
+	for devID, driver := range qatDeviceDriver {
+		for _, enabledDriver := range dp.kernelVfDrivers {
+			if driver != enabledDriver {
+				continue
+			}
+
+			err := writeToDriver(filepath.Join(dp.pciDriverDir, dp.dpdkDriver, "new_id"), vendorPrefix+devID)
+			if err != nil && !errors.Is(err, os.ErrExist) {
+				return errors.Wrapf(err, "failed to set device ID %s for %s. Driver module not loaded?", devID, dp.dpdkDriver)
```

Comment:
nit: the error contains a trace stack already, no need to add one more
```suggestion
				return errors.WithMessagef(err, "failed to set device ID %s for %s. Driver module not loaded?", devID, dp.dpdkDriver)
```